mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/dao/DaoDeleteMethodGenerator.java

```diff
@@ -164,7 +174,13 @@ public DaoDeleteMethodGenerator(
         "$T boundStatementBuilder = $L.boundStatementBuilder()",
         BoundStatementBuilder.class,
         statementName);
-
+    if (runtTimeAttributeParam != null) {
+      if (runtTimeAttributeParam != null) {
```

Comment:
why two ifs?

Reply:
because I'm a dope. Thanks I'll remove this.

Reply:
Still present.